### DISCUSSION TITLE
Use a 3rd-party feedback widget

### DISCUSSION THREAD

**Discussion by klochowicz** [get10101/itchysats#1933] [Ideas]
before we venture into the territory of creating in-house messaging platform or user registration, I thought we could explore some ready-made solutions for react to save us time and allow collecting feedback sooner.

I had a look at 2 platforms that give us ready-made feedback widgets:
- Feedback Fish: https://feedback.fish/get-started/
- SerVoice: https://www.servoice.io/

Feedback Fish seems to be really appealing, and it's free to try it out (first 25 feedbacks). then it's ~$20 per month in case we'd like to continue using it.

As they have a react component ready, pretty much all code that needs to be into the app is trivial: https://feedback.fish/help/react

I'd imagine that feedback is something more critical on early stage, so even if we start paying, we could stop in a couple of months. And if we don't get feedbacks, we won't pay anything.

I'd like to go ahead with this as it's super easy to implement and from briefly looking at the react component it's super easy (looks like we can correlate feedback to a `taker_id`, which might help whether feedback comes from people actually opening positions`).
There's a way of passing in emails (opt-in), in case we'd like to respond to user feedback.

**Comment by luckysori** [comment]
If there is a low effort way to let people that are actually opening the app give us feedback, I think we should do it!

Alternatively we could link to GitHub, but not all feedback belongs there and the user would need a feedback account.

**Reply by klochowicz** [reply]
we do link to github at the bottom of the page, but I guess not everyone is motivated enough to add anything (TBF the link points to the main repo, not to `issues` or `discussions`) . making/using a github account gets rid of a lot of anonymity perhaps, or maybe it's just a bit too much hassle. After testing the UX of this one, it seemed really nice & easy.

I figured out that this could be an easy way of trying it out - and if we won't get 25 feedbacks, we will never pay anyway 😀

**Comment by bonomat** [comment]
A ticket was created: #1936 marking this as closed.
